src/util/color/predefinedcolorpalettes.cpp

```diff
@@ -124,6 +123,22 @@ const ColorPalette PredefinedColorPalettes::kMixxxHotcueColorPalette =
                         kColorMixxxWhite,
                 });
 
+const ColorPalette PredefinedColorPalettes::kMixxxHotcueColorPaletteWithDefaultColor =
+        ColorPalette(
+                QStringLiteral("Mixxx Hotcue Colors (with default color)"),
+                QList<mixxx::RgbColor>{
+                        kColorMixxxRed,
+                        kColorMixxxYellow,
+                        kColorMixxxGreen,
+                        kColorMixxxCeleste,
+                        kColorMixxxBlue,
+                        kColorMixxxPurple,
+                        kColorMixxxPink,
+                        kColorMixxxWhite,
+                        kSchemaMigrationReplacementColor,
+                },
+                QList<unsigned int>{8});
+
```

Comment:
What are the use cases for kMixxxHotcueColorPaletteWithDefaultColor and kMixxxHotcueColorPalette? 
How about adding orange to the kMixxxHotcueColorPalette?